backend/src/kitconcept/intranet/profiles/default/types/Person.xml

```diff
@@ -5,8 +5,22 @@
         i18n:domain="kitconcept.intranet"
 >
   <property name="behaviors"
-            purge="false"
+            purge="true"
   >
+    <element value="kitconcept.core.person_image" />
+    <element value="volto.kicker" />
+    <element value="collective.person.person" />
+    <element value="kitconcept.intranet.person" />
+    <element value="collective.contact_behaviors.contact_info" />
+    <element value="kitconcept.core.additional_contact_info" />
+    <element value="plone.namefromtitle" />
+    <element value="plone.shortname" />
+    <element value="volto.navtitle" />
+    <element value="plone.excludefromnavigation" />
```

Comment:
Person is not enabled in the Navigation control panel, so I don't think these are relevant.
```suggestion
```

Reply:
These were set in kitconcept.core, and I decided not to change. Also, as a rule of thumb, I always add `plone.excludefromnavigation` because if an admin decides this content type should appear on the navigation and the behavior is not present, no instances will ever be displayed.